### PR TITLE
Rename methods name

### PR DESCRIPTION
**Pull request by boompttn**

### CODE REVIEW

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ jobs:
         with:
           fetch-depth: 0
       - run: |
+          brew install tree
```
Comment (new5558) [inline .github/workflows/pull-request.yaml:14]: ??

File: proto/hts/participant/service.proto [hunk 2]
```diff
@@ -52,28 +52,44 @@ message IsEventAvailableRequest {
 }
 
 service ParticipantService {
+    // Check whether event's start date is before specified date
```
Comment (new5558) [inline proto/hts/participant/service.proto:55]: So we now need to comment on each method?

**Review by new5558 — APPROVED** [review]
LGTM but please look at comments

File: proto/hts/participant/service.proto (hunk 2 above)
Comment (boompttn) [reply proto/hts/participant/service.proto:55]: Heard Five talk about it

File: .github/workflows/pull-request.yaml (hunk 1 above)
Comment (boompttn) [reply .github/workflows/pull-request.yaml:14]: tree command not found, have to brew install tree

**Pull request merged**